fiftyone/core/utils.py

```diff
@@ -2294,6 +2292,10 @@ def recommend_thread_pool_workers(num_workers=None):
 def recommend_process_pool_workers(num_workers=None):
     """Recommends a number of workers for a process pool.
 
+    By default, ``multiprocessing.cpu_count()`` workers will be recommended if
+    you are running on macOS or Linux, while a single worker will be
+    recommended on Windows.
```

Comment:
TODO: is multiprocessing on Windows more stable now?

Can we use `num_workers=multiprocessing.cpu_count()` by default instead of `num_workers=1`?

Reply:
Crated a ticket to track this, will test tomorrow

Reply:
Tested on windows, multiprocessing does work for me, didn't run into any issue except the initial fiftyone setup.